src/Makefile

```diff
@@ -277,7 +277,7 @@ mpi-stubs:
 sinclude ../lib/python/Makefile.lammps
 install-python:
 	@$(PYTHON) ../python/install.py -v ../src/version.h \
-		-m ../python/lammps.py -l ../src/liblammps.so
+		-m ../python/lammps.py -l ../src/liblammps.*[dylib,so]
```

Comment:
This cannot be right. This will look for a file starting with ../src/liblammps. followed by any number of characters and then terminated by either 'd', 'y', 'l', 'i', 'b', ',', 's', or 'o'. That does match both, ../src/liblammps.dylib and ../src/liblammps.so, but not exclusively one of those two.

Reply:
I am working on this